Common++/header/LRUList.h

```diff
@@ -115,26 +102,22 @@ namespace pcpp
 			m_CacheItemsMap.erase(iter);
 		}
 
-		/**
-		 * @return The max size of this list as determined in the c'tor
-		 */
-		size_t getMaxSize() const
+		/// @return The max size of this list as determined in the c'tor
+		std::size_t getMaxSize() const
 		{
 			return m_MaxSize;
 		}
 
-		/**
-		 * @return The number of elements currently in this list
-		 */
-		size_t getSize() const
+		/// @return The number of elements currently in this list
+		std::size_t getSize() const
 		{
 			return m_CacheItemsMap.size();
 		}
 
 	private:
 		std::list<T> m_CacheItemsList;
 		std::unordered_map<T, ListIterator> m_CacheItemsMap;
-		size_t m_MaxSize;
+		std::size_t m_MaxSize;
```

Comment:
Why replacing `size_t` with `std::size_t`? 🤔

Reply:
If you look at the CI fails from the first 2 commits in this PR, they fail because of unrecoginized `size_t` and `string`.

Reply:
These are all `clang-tidy` errors 😕 
It's probably because these files haven't been changed in a long time. Maybe we should suppress this error? We don't use `std::size_t` anywhere else and it complies and runs ok...

Reply:
If I'm not mistaken including `cstddef` fixes it. Just clang-tidy can't find definition

Reply:
I prefer the error be fixed instead of suppressed, but I am fine either way.
I usually prefer using `std::size_t` in C++ due to the standard containers using it and usually defining it.

Both `size_t` and `std::size_t` alias to the same type. The slight difference is that `size_t` is from the C runtime and `std::size_t` is from the C++ runtime, and its a bit iffy which of the 2 are defined where.

For instance:
- when using CPP style headers `<cstring>`, `<cstddef>`, etc `std::size_t` is defined.
- when using C style headers `<string.h>`, `<stddef.h>` `size_t` is defined.

Reply:
It's just that we're using `size_t` everywhere else, so it's more consistent

Reply:
Sure

Reply:
Changed it back to `size_t` 
72dd3b3